src/stores/subscribable.ts

```diff
@@ -1,8 +1,8 @@
 import { browser } from '$app/environment';
 import { env } from '$env/dynamic/public';
 import { createClient, type Client, type ClientOptions } from 'graphql-ws';
-import { isEqual } from 'lodash-es';
-import type { Readable, Subscriber, Unsubscriber, Updater } from 'svelte/store';
+import { debounce, isEqual } from 'lodash-es';
+import { type Readable, type Subscriber, type Unsubscriber, type Updater } from 'svelte/store';
```

Comment:
Do these specifically need to have `type` before them instead?

Reply:
Weird, prettier must have auto formatted this. Let me see if I can revert it.

Reply:
Done